sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -38,16 +40,20 @@ pub struct InputSelection {
     required_inputs: HashSet<OutputId>,
     forbidden_inputs: HashSet<OutputId>,
     selected_inputs: Vec<InputSigningData>,
+    context_inputs: HashSet<ContextInput>,
     outputs: Vec<Output>,
     addresses: HashSet<Address>,
     burn: Option<Burn>,
     remainder_address: Option<Address>,
-    protocol_parameters: ProtocolParameters,
-    slot_index: SlotIndex,
+    slot_commitment_id: SlotCommitmentId,
     requirements: Vec<Requirement>,
     automatically_transitioned: HashSet<ChainId>,
-    mana_allotments: u64,
+    issuer_id: AccountId,
```

Comment:
Would make this optional for cases where someone else issues the block or one has already still a lot BIC and doesn't want more (so no automatic allotment, if None is provided)

Reply:
it is done